.github/workflows/jellyfish-tests.yml

```diff
@@ -0,0 +1,52 @@
+name: Jellyfish Tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f
+        with:
+          repository: DeFiCh/jellyfish
+          ref: 'monstrobishi/specify-image'
+
+      - uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f
+        with:
+          path: defichain
+
+      - uses: actions/setup-node@46071b5c7a2e0c34e49c3cb8a0e792e86e18d5ea
+        with:
+          node-version: '15'
+
+      - uses: actions/cache@v2
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+
+      - name: Set build version
+        run: |
+          cd defichain
+          TARGETS="x86_64-pc-linux-gnu" ./make.sh git_version
+              
+      - name: Build and package
+        run: |
+          echo ">>> BUILD VERSION <<<"
+          echo ${BUILD_VERSION}
+          cd defichain
+          TARGETS="x86_64-pc-linux-gnu" ./make.sh docker-release-git
+          docker tag defichain-x86_64-pc-linux-gnu:$BUILD_VERSION defichain-x86_64-pc-linux-gnu:dockerhub-latest
+          docker build -t test-build-container -f ./contrib/dockerfiles/dockerhub/x86_64-pc-linux-gnu.dockerfile .
```

Comment:
You could chain this to run after dev-builds.yml has finished since you don't need to build this again.

Reply:
Yeah I had this thought too, best to leave for a future improvement